src/pages/welcome-offer.jsx

```diff
@@ -0,0 +1,58 @@
+const features = [
+  {
+    image: "/src/assets/images/icons/rs-50-cashbak.png",
+    heading: " Select where to earn 2% cashback",
+    subheading: "Choose either shopping, dining, or travel for 2% cashback",
+  },
+  {
+    image: "/src/assets/images/icons/edge-card-bill-xxxl.png",
+    heading: " Pay with UPI or credit card",
+    subheading:
+      "Earn 2% cashback on your selected category and 0.4% on all other spends",
+  },
+  {
+    image: "/src/assets/images/icons/edge-card-bill-xx-xl.png",
+    heading: " Switch with 1-tap",
+    subheading: "You can switch to a different category every 3 billing cycles",
+  },
+];
+export default function WelcomeOffer() {
+  return (
+    <>
+      <main className="">
+        <div className="max-w-md mx-auto ">
+          <section className="pt-24 px-4 bg-accent-primary">
+            <div className="mb-9">
+              <h2 className="text-[32px] font-medium leading-[40px] text-white mb-2.5">
```

Comment:
please add this font styling in the config.js file. dont use text-[32px] like this. also use leading-10 for 40px line hight